src/mainnet-nft-list.json

```diff
@@ -42,5 +42,9 @@
   {
     "name": "CyberBoxCertNFT",
     "contractAddress": "0xa4a8e345e1a88efc9164014bb2cebd4c2f98e986"
+  },
+  {
+    "name": "ValoraPunks",
+    "contractAddress": "0xaF99AC3aB1c17728d0Ee3256f02061018845b588"
```

Comment:
this has some upper case characters, not sure if that's ok

Reply:
if it's not we should add test coverage for that

Reply:
I added a test for address upper casing to valora-rest-api that can be imitated https://github.com/valora-inc/valora-rest-api/pull/228/files

Reply:
oh looks like other addresses in this file are mixed case too

Reply:
It's ok.  We already have an annoying lack of checksummed and lowercased contract addresses.  I've meant to add a CI job to force one or the other.